### ISSUE TITLE
Deleting a Campaign deck throws error

### ISSUE THREAD

**Issue by mattsigal** [Kamalisk/arkhamdb#92]
Tried to delete a deck today from my account (so a deck that has had upgrades applied). I checked the deck and went "Delete all decks selected". After the popup to confirm the deletion, an error gets thrown:

![screen shot 2017-05-12 at 16 49 35](https://cloud.githubusercontent.com/assets/1421758/26016444/0281cd7e-3733-11e7-8467-4a424a26b08a.png)

If I go in and delete each upgrade individually, it works fine. So it appears the delete function is not currently allowing for deleting campaign decks that have multiple versions.  Hopefully this is an easy fix!

**Comment by Kamalisk** [comment]
That is slightly intentional, it should give a better warning or add a seperate option to delete campaign decks.

**Comment by mattsigal** [comment]
Yeah, I understand that reasoning - although I think that we have so many ways of deleting decks, at least one should allow for deleting deck stacks that have upgrades.

I think this makes the most sense:

![screen shot 2017-05-16 at 12 06 10 pm](https://cloud.githubusercontent.com/assets/1421758/26116209/4a2dcc04-3a30-11e7-8e84-c4e18cc9238c.png)

And in the warning that pops up from "Delete all decks selected" it should reiterate that this would delete all previous versions of the selected decks as well. In that dropdown, there could be another option ("Delete all top-level decks from selected" or something), if you want to give users that option?
